<?xml version="1.0"?>
<launch>

  <arg name="robot_name"      default="r0" />
  <arg name="global_frame_id" default="map" />
  <arg name="path_file"       default="" />
  <arg name="min_waypoint_distance"       default="0.1" />  <!-- on 0 it uses all waypoints to create a spline /-->
  
  <!-- arg name="path_file"       default="$(find tuw_waypoint_to_spline)/cfg/route1_x1y0a0.yaml" /-->
  
  <group ns="$(arg robot_name)">
    <node pkg="tuw_waypoint_to_spline" type="tuw_waypoint_to_spline_node" name="tuw_waypoint_to_spline_node" output="screen">
        <param name="global_frame" value="$(arg global_frame_id)" />
        <param name="path_file" value="$(arg path_file)" />
        <param name="min_waypoint_distance" value="$(arg min_waypoint_distance)" />
        <remap from="path" to="global_planner/planner/plan" />
    </node>
  </group>
</launch>
